pkg/kafka/policy.go

```diff
@@ -183,13 +233,12 @@ func (req *RequestMessage) ruleMatches(rule api.PortRuleKafka) bool {
 		return false
 	}
 
-	flowdebug.Log(log.WithFields(logrus.Fields{
-		fieldRequest: req.String(),
-		fieldRule:    rule,
-	}), "Matching Kafka rule")
+	if rule.APIKey != "" && rule.Role != "" {
```

Comment:
Why are we not rejecting this in the validation code?

Reply:
I will move it to `Sanitize`

Reply:
Don't forget the CRD validation please

Reply:
I need more context here sorry. How do I do CRD validation? I have already run it with `make generate-k8s-api`. Anything else needed?

Reply:
We already do this check in `Sanitize`. removing this check here